src/nvim/eval.c

```diff
@@ -17037,18 +17037,6 @@ long get_vim_var_nr(int idx) FUNC_ATTR_PURE
   return vimvars[idx].vv_nr;
 }
 
-/// Get typval_T representing v: variable
-///
-/// @warning if v: variable has reference counter it is not increased.
-///
-/// @param[in]  idx  Variable index, @see VimVarIndex.
-typval_T get_vim_var_tv(VimVarIndex idx) FUNC_ATTR_PURE
```

Comment:
@justinmk This function was not ever used for anything, but special variables (v:true, v:false, v:null). So note about reference counter is not applicable anywhere where function was used. There is also no need to have such indirection, there are not too much places where function is needed.